soda_machine.rb

```diff
@@ -7,12 +9,31 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length # or .size
   end
 
   def find_soda(soda_brand)
+    @sodas.find {|soda| soda.brand == soda_brand}
   end
 
   def sell(soda_brand)
```

Comment:
This is good code. Nice work. But why create an `@sold_soda` instance variable? Unnecessary. Users of the `SodaMachine` class should get sodas from the `sell` method. Why do we need to keep track of the result of `sell` inside the `SodaMachine` class.